paper.rst

```diff
@@ -883,22 +883,22 @@ Python/NumPy `scipy.signal.welch()` function to use the array API.
 
 The resulting code can now be run against any array API conforming library.
 `Fig. 2`_ shows the resulting speedups vs. NumPy for
-`LinearDiscriminantAnalysis` and `scipy.signal.welch()` on Torch CPU, Torch
+`LinearDiscriminantAnalysis` and `scipy.signal.welch()` on PyTorch CPU, PyTorch
 GPU (CUDA), and CuPy backends. GPU backends give a significant speedup, but
-even Torch CPU can have up to 2x speedup over NumPy.
+even PyTorch CPU can have up to 40% speedup over NumPy.
 
 `Fig. 2`_ additionally highlights an additional type of change, namely
 **making use of library specific performance optimizations**. The SciPy
 `welch()` implementation uses an optimization involving stride tricks. Stride
 tricks have not been standardized in the array API since they are not
-available in some libraries (e.g., JAX). NumPy, CuPy, and Torch allow setting
+available in some libraries (e.g., JAX). NumPy, CuPy, and PyTorch allow setting
 strides, but they do not use a uniform API to do so. An array API compatible
 implementation can be used, but it is slower, so it is used only as a fallback
 for libraries outside of NumPy, PyTorch, and CuPy. Indeed, it is significantly
-slower than even plain NumPy, with PyTorch CUDA taking 200 seconds to
+slower than even plain NumPy, with PyTorch GPU taking 200 seconds to
```

Comment:
If thus language is preferred it should be updated in the figure too.

Reply:
Ah, I thought the inconsistency I caught was only in the text, didn't pay attention to the plots 🤦🏻‍♂️ Yes, we should unify them.

Reply:
I am looking at the latest main branch, and noticed that all references to "PyTorch CUDA" are gone. I'll update the figure.

Reply:
Done in 6f20704.